packages/autorest.go/test/autorest/paginggroup/paginggroup_test.go

```diff
@@ -38,6 +39,23 @@ func httpClientWithCookieJar() policy.Transporter {
 	return http.DefaultClient
 }
 
+func TestAppendAPIVersion(t *testing.T) {
+	t.Skip("Go doesn't append api-version")
```

Comment:
Should we have this test then?

Reply:
I wanted to be explicit that we're skipping this test (and others) and didn't just overlook them.